.travis.yml

```diff
@@ -7,36 +7,41 @@ python: 2.7
 cache: false
 
 env:
-  - TOX_ENV=pypy-twisted_14.0-pyopenssl_0.13.1
-  - TOX_ENV=pypy-twisted_14.0-pyopenssl_0.14
-  - TOX_ENV=pypy-twisted_14.0-pyopenssl_0.15.1
-  - TOX_ENV=pypy-twisted_15.1-pyopenssl_0.13.1
-  - TOX_ENV=pypy-twisted_15.1-pyopenssl_0.14
-  - TOX_ENV=pypy-twisted_15.1-pyopenssl_0.15.1
-  - TOX_ENV=pypy-twisted_15.3-pyopenssl_0.13.1
-  - TOX_ENV=pypy-twisted_15.3-pyopenssl_0.14
-  - TOX_ENV=pypy-twisted_15.3-pyopenssl_0.15.1
-  - TOX_ENV=pypy-twisted_15.5-pyopenssl_0.13.1
-  - TOX_ENV=pypy-twisted_15.5-pyopenssl_0.14
-  - TOX_ENV=pypy-twisted_15.5-pyopenssl_0.15.1
-  - TOX_ENV=py27-twisted_14.0-pyopenssl_0.13.1
-  - TOX_ENV=py27-twisted_14.0-pyopenssl_0.14
-  - TOX_ENV=py27-twisted_14.0-pyopenssl_0.15.1
-  - TOX_ENV=py27-twisted_15.1-pyopenssl_0.13.1
-  - TOX_ENV=py27-twisted_15.1-pyopenssl_0.14
-  - TOX_ENV=py27-twisted_15.1-pyopenssl_0.15.1
-  - TOX_ENV=py27-twisted_15.3-pyopenssl_0.13.1
-  - TOX_ENV=py27-twisted_15.3-pyopenssl_0.14
-  - TOX_ENV=py27-twisted_15.3-pyopenssl_0.15.1
-  - TOX_ENV=py27-twisted_15.5-pyopenssl_0.13.1
-  - TOX_ENV=py27-twisted_15.5-pyopenssl_0.14
-  - TOX_ENV=py27-twisted_15.5-pyopenssl_0.15.1
-  - TOX_ENV=py33-twisted_15.5-pyopenssl_0.15.1
-  - TOX_ENV=py34-twisted_15.5-pyopenssl_0.15.1
+  - TOX_ENV=pypy-twisted_16.0-pyopenssl_0.13.1
```

Comment:
We may want to consider cutting down on older pyOpenSSLs at some point soon.  Cryptography is unambiguously the future.